Extension/Global/Strings.cs

```diff
@@ -24,6 +24,12 @@ public static class Strings
         public const string SETTINGS_PREFER_SHIELD_TOOLTIP = "Prefer infantry with shields when upgrading to infantry troop classes.";
         public const string SETTINGS_RANGED_PREFERENCE_NAME = "Prefer ranged with specific weapons";
         public const string SETTINGS_RANGED_PREFERENCE_TOOLTIP = "Prefer specific ranged weapons when upgrading to ranged troop classes. (0 for no preference, 1 for Bows, 2 for crossbows)";
+        public const string SETTINGS_BUFFER_DAILY_COST_NAME = "Keep enough gold for X days of troop wages?";
+        public const string SETTINGS_BUFFER_DAILY_COST_TOOLTIP = "Reserves gold to pay daily costs for X amount of days.";
+        public const string SETTINGS_SPLIT_UPGRADES_NAME = "Split upgrades if decision can't be made";
+        public const string SETTINGS_SPLIT_UPGRADES_TOOLTIP = "Splits upgradepaths when no decision can be made. NOTE: decisions will be made based on culture strengths.";
+        public const string SETTINGS_PLAYER_DECISION_NAME = "Lets player decide on every upgrade split path";
```

Comment:
If I've interpreted this right, this disables multipath decision making correct? So behaviour with this enabled is single path troops are upgraded on upgrade all, but multipath are skipped?
And disabled all upgrades are resolved based on other settings?

Reply:
Playerdecision will only do the upgrades that do not branch, where the player will need to make a decision